server/init.go

```diff
@@ -39,10 +40,36 @@ type GenesisTx struct {
 }
 
 var (
-	flagOverwrite = "overwrite"
-	flagGenTxs    = "gen-txs"
-	flagIP        = "ip"
-	flagChainID   = "chain-id"
+	//--name parameter name, init gen-tx command
+	FlagName = "name"
+	//--home-client parameter name, init gen-tx command
+	FlagClientHome = "home-client"
+	//--owk parameter name, init gen-tx command
+	FlagOWK = "owk"
+
+	// bonded tokens given to genesis validators/accounts
+	FreeFermionVal = int64(100)
+	// ...
+	FreeFermionsAcc = int64(50)
```

Comment:
remove global variables, pass these directly to the functions which are using these variables. Also pass in the Denom, we should not assume a denom of `fermions`